### DISCUSSION TITLE
change indicator next to line numbers

### DISCUSSION THREAD

**Discussion by mzagrabe** [helix-editor/helix#11039] [Q&A] [answered]
Greetings,

I'm new to helix. I'm running through the `--tutor` and there are some helpful indicators next to the line numbers when I modify a line or add a new line. When I open other files (and make changes) I don't see these vertical indicator characters. What are they called? and how do I get them to show up besides in the `--tutor`?

Thanks for any help!

**Comment by gyreas** [comment, marked as answer]
Are they colored bars? If yes, they're git status indicator for that file. Only files in a git index can trigger them.

You're probably seeing them because the actual tutor file is still linked to the local helix clone/fork

**Reply by mzagrabe** [reply]
Thanks for the reply @gyreas. That looks to be the case. Color me informed.
